android/app/src/main/java/org/fossasia/openevent/core/speaker/SpeakerViewHolder.java

```diff
@@ -105,9 +106,7 @@ public void bindSpeaker(Speaker speaker) {
                         // bitmap has dark muted swatch profile
                         speakerTextualInfo.setBackgroundColor(swatch2.getRgb());
                     } else {
-                        // Default color is red if swatches cannot be found
-                        int defaultColor = 0xFFb20000;
-                        speakerTextualInfo.setBackgroundColor(defaultColor);
+                        speakerTextualInfo.setBackgroundColor(ContextCompat.getColor(context, R.color.speakers_viewholder_background));
```

Comment:
No, please use one of the theme colors. It is customisable through the app generator

Reply:
Oh, thanks for the clarification!